### ISSUE TITLE
Add image to Docker Hub

### ISSUE THREAD

**Issue by AuthorShin** [devfake/flox#192]
Hello
when will you consider adding the docker image to hub.docker.com?

**Comment by YodaQ** [comment]
I agree that would be great and make it much easier to install and manage

**Comment by antgel** [comment]
I guess the bigger question is where are you? It seems to be awfully quiet around here.

**Comment by AuthorShin** [comment]
Yeah totally agree, it's very unfortunate to see such a great project like this left alone with no care.
It's literally one of the nicest and handful projects that I've seen in self hosted section of the GitHub.

**Comment by AuthorShin** [comment]
Hey man @devfake
